wlauto/workloads/gmail/__init__.py

```diff
@@ -0,0 +1,79 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from wlauto import AndroidUxPerfWorkload, Parameter, File
+from wlauto.exceptions import ValidationError
+from wlauto.utils.types import list_of_strings
+from wlauto.utils.misc import unique
+
+
+class Gmail(AndroidUxPerfWorkload):
+
+    name = 'gmail'
+    package = 'com.google.android.gm'
+    min_apk_version = '6.7.128801648'
+    activity = ''
+    view = [package + '/com.google.android.gm.ConversationListActivityGmail',
+            package + '/com.google.android.gm.ComposeActivityGmail']
+    description = """
+    A workload to perform standard productivity tasks within Gmail.  The workload carries out
+    various tasks, such as creating new emails, attaching images and sending them.
+
+    Test description:
+    1. Open Gmail application
+    2. Click to create New mail
+    3. Attach an image from the local Images folder to the email
+    4. Enter recipient details in the To: field
+    5. Enter text in the Subject edit box
+    6. Enter text in the Compose edit box
+    7. Click the Send mail button
+    """
+
+    parameters = [
+        Parameter('recipient', kind=str, mandatory=True,
+                  description=""""
+                  The email address of the recipient.  Setting a void address
+                  will stop any mesage failures clogging up your device inbox
+                  """),
+        Parameter('test_image', kind=str, mandatory=True, default='uxperf_1600x1200.jpg',
```

Comment:
I copy and pasted from previously accepted workloads
See GoogleSlides and Skype.